server/container/util/src/test/java/org/apache/james/util/RunnablesTest.java

```diff
@@ -0,0 +1,40 @@
+package org.apache.james.util;
```

Comment:
licence

Reply:
Still no license

Reply:
I don't understand how it happened because I added it ... now fixed

Reply:
héhé you know git sometime is tricky ^^